### ISSUE TITLE
Report qqplot by MAF

### ISSUE THREAD

**Issue by filosi** [genepi/nf-gwas#38]
Provide qqplot in the report divided by maf

**Comment by filosi** [comment]
Yes, I do have but it's missing the point optimization you use in the `ramwas` - `qqPlotFast` function.

**Comment by filosi** [comment]
I added the code snipped in the RMD report in the pull request  #41 . It does not use the `qqPlotPrepare` function and there is no estimation of the CI, we can add it later if you think that is worth it.
The default used 4 MAF interval (if any variant falls in the interval) respectively:
- (0, 0.0005]
- (0.0005, 0.005]
- (0.005, 0.05]
- (0.05, 1]
This behavior can change in the future since the function takes the intervals as argument.

**Comment by seppinho** [comment]
Awesome, can you create a pull request including just the QQ-plot adaption? At the moment the pull request also includes the gene-based commits.

**Comment by filosi** [comment]
Yes, right! Then remove the #41 and I'll do it again!
